app/views/idv/how_to_verify/show.html.erb

```diff
@@ -11,57 +11,67 @@
     ) do |f|
 %>
   <div role="group" aria-labelledby="how-to-verify-info">
-    <%= f.radio_button(
-          :selection,
-          Idv::HowToVerifyForm::REMOTE,
-          class: 'usa-radio__input usa-radio__input--tile',
+    <%= f.hidden_field(
+          :selection, :value => Idv::HowToVerifyForm::REMOTE,
```

Comment:
Our codebase uses new-style hashes, not hashrockets when we can avoid them
```suggestion
          :selection, value: Idv::HowToVerifyForm::REMOTE,
```

Reply:
We found an inconsistency in the original  Spanish gengo translation:

> _"Tomará fotografías de **tu** identificación para verificar su identidad completamente en línea. La mayoría de los usuarios finalizan este proceso de una sola vez. "_

should read...
> _"Tomará fotografías de **su** identificación para verificar su identidad completamente en línea. La mayoría de los usuarios finalizan este proceso de una sola vez. "_